app/components/MediaItemCard/tests/index.test.js

```diff
@@ -0,0 +1,46 @@
+/**
+ *
+ * Tests for MediaItemCard
+ *
+ */
+
+import React from 'react';
+import { renderWithIntl } from '@utils/testUtils';
+import MediaItemCard from "../index";
+
+describe('<MediaItemCard />', () => {
```

Comment:
Can we please test for all branches here as well, because we are adding if conditions. Every else case needs to be tested as well